public/js/cohortFilter.js

```diff
@@ -0,0 +1,47 @@
+let tableRows = document.querySelectorAll('.table-row')
+let applicantsDates = document.querySelectorAll('.applicants-cohort-date')
+
+/**
+ * filter through nodelist of cohort date items to hide mismatches
+ *
+ * @param cohortDate
+ */
+function applyCohortFilter(cohortDate) {
+    let mismatches = [];
+    applicantsDates.forEach(function (date, index) {
+        let theDates = date.getAttribute('data-id')
+        if(cohortDate !== theDates) {
+            let row = date.parentNode
+            mismatches.push(row)
```

Comment:
what is mismatches for? just do `row.classList.add('hidden')`

Reply:
👍